PennMobile/Home/Controllers/HomeViewController.swift

```diff
@@ -68,7 +68,13 @@ class HomeViewController: GenericViewController {
             let firstName = Account.getAccount()?.first ?? GSRUser.getUser()?.firstName
             if let firstName = firstName {
                 let intros = ["Welcome", "Howdy", "Hi there", "Hello", "Greetings", "Sup"]
-                self.displayTitle = "\(intros.random!), \(firstName)!"
+                
+                if firstName != "" {
+                    self.displayTitle = "\(intros.random!), \(firstName)!"
+                } else {
+                    self.displayTitle = "Howdy!"
+                }
+                
                 titleCacheTimestamp = Date()
             } else {
                 self.displayTitle = "Home"
```

Comment:
I think we should display the same message for users, who are not signed in, & alumni. Both should say "home" or both should give a customized greeting like "howdy." 

I like the latter option more, what do you think?